### DISCUSSION TITLE
已交费未能成功续期

### DISCUSSION THREAD

**Discussion by Willbill202** [getlantern/lantern-forum-cn#974] [Q&A]
10月20日完成交费，续期两年，但蓝灯始终未完成续期，email: widalzq@gmail.com,请处理。

**Comment by MaxMKMax** [comment]
@Willbill202 您好. 请您提供我支付单据. 然后我再帮您提交问题上去. 谢谢.

**Reply by Willbill202** [reply]
![Screenshot_20241020-093003](https://github.com/user-attachments/assets/cc7971dd-34a7-4f69-a082-b19f3c5268b2)

**Reply by Willbill202** [reply]
@Mike777Khoo Mike，已经提交了付款凭证，请帮忙申请，谢谢。

**Reply by Willbill202** [reply]
@Mike777Khoo Mike，问题还是没得到解决，专业版到期日还是没续费前的日期，您帮我提交问题了吗?

**Comment by MaxMKMax** [comment]
@Willbill202 您好. 已经帮您提交问题上去了. 让我再帮您检查一下或者重新帮您提交. 谢谢您的耐心等待!

**Comment by MaxMKMax** [comment]
@Willbill202 已经帮您再次提交好了. 谢谢您支持蓝灯!
